src/js/lib/youtube.js

```diff
@@ -21,11 +21,13 @@ export function pimpYouTubePlayer(videoId, node, height, width, chapters) {
 
                             const playerTotalTime = youTubePlayer.getDuration();
                             chapTimer = setInterval(function() {
+                                let chapterCurrentProgress;
```

Comment:
I think it's the latest I can declare it — needs to be accessible in lines 30 and 41.
Is there another place I could declare it?